app/src/main/java/com/daotrung/mynote/ui/update/UpdateFragment.kt

```diff
@@ -0,0 +1,58 @@
+package com.daotrung.mynote.ui.update
+
+import android.os.Bundle
+import android.text.TextUtils
+import androidx.fragment.app.Fragment
+import android.view.LayoutInflater
+import android.view.View
+import android.view.ViewGroup
+import android.widget.Toast
+import com.daotrung.mynote.R
+import androidx.fragment.app.viewModels
+import androidx.navigation.fragment.findNavController
+import com.daotrung.mynote.database.TaskEntry
+import com.daotrung.mynote.databinding.FragmentUpdateBinding
+import com.daotrung.mynote.viewmodel.TaskViewModel
+
+class UpdateFragment : Fragment() {
+
+    private val viewModel : TaskViewModel by viewModels()
+    private var _binding: FragmentUpdateBinding? = null
+
+    private val binding get() = _binding!!
+    override fun onCreateView(
+        inflater: LayoutInflater, container: ViewGroup?,
+        savedInstanceState: Bundle?
+    ): View? {
+        _binding = FragmentUpdateBinding.inflate(inflater,container,false)
+        val args = UpdateFragmentArgs.fromBundle(requireArguments())
+
+        binding.apply {
+            updateEdtTask.setText(args.taskEntry.title)
+            updateSpinner.setSelection(args.taskEntry.priority)
+
+            btnUpdate.setOnClickListener {
```

Comment:
sự kiện onClick em nên impelemnt ở Fragment 
```suggestion
            btnUpdate.setOnClickListener (this)
```